movieApp.css

```diff
@@ -0,0 +1,50 @@
+@media only screen and (max-width: 600px) {
```

Comment:
You got the styling down which is nice. It is done in accordance with the rubric.